lib/ui/home/widget/home_survey_page.dart

```diff
@@ -1,26 +1,27 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/model/survey_model.dart';
 import 'package:flutter_survey/theme/app_dimensions.dart';
 import 'package:flutter_survey/ui/widget/dimmed_background.dart';
 import 'package:flutter_survey/ui/widget/next_button.dart';
 
 const _maxLines = 2;
 const _descriptionOpacity = 0.7;
 
-// TODO: Integrate surveys https://github.com/nimblehq/ic-flutter-taher-toby/issues/13
 class HomeSurveyPage extends StatelessWidget {
+  final SurveyModel survey;
   final VoidCallback onNextButtonPressed;
 
   const HomeSurveyPage({
     Key? key,
+    required this.survey,
     required this.onNextButtonPressed,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Stack(
       children: [
-        DimmedBackground(background: Assets.images.dummyBackground.path),
+        DimmedBackground(background: survey.coverImageUrl),
```

Comment:
Should we use hd image?

Reply:
This is already the case, it's being mapped in `survey_model.dart` - `getHdCoverImageUrl()`